.github/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build Docker Image
```

Comment:
we build the image  in the repo:[ratehub-gateway ](https://github.com/ratehub/ratehub-gateway)

Express gateway is just NPM package, we don't need to build the image here, we need to tag with semantic release and npm pack and push to package cloud: 

A combination of the following will have to be done - 
https://github.com/ratehub/event-service/blob/master/.github/workflows/publish.yml

and this, although this is in a monorepo and you won't use lerna for a single package - 
https://github.com/ratehub/nodejs/blob/master/.github/workflows/release.yml

But just as reference